zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/UserTaskController.java

```diff
@@ -87,6 +90,27 @@ public CompletableFuture<ResponseEntity<Object>> assignUserTask(
                 .orElseGet(() -> ResponseEntity.noContent().build()));
   }
 
+  @PatchMapping(
+      path = "/user-tasks/{userTaskKey}",
+      produces = "application/json",
+      consumes = "application/json")
+  public CompletableFuture<ResponseEntity<Object>> updateUserTask(
+      final ServerWebExchange context,
+      @PathVariable final long userTaskKey,
+      @RequestBody(required = false) final UserTaskUpdateRequest updateRequest) {
+
```

Comment:
💭 We are not enforcing the request body here, just wondering if we should (as PATCH is a partial update operation)

Reply:
I kept this to rather validate the request body manually for being empty now. This provides the same response consistently for missing update data.